package.json

```diff
@@ -43,26 +43,27 @@
   },
   "homepage": "https://github.com/mostjs/dom-event#readme",
   "devDependencies": {
-    "@most/eslint-config-most": "^1.0.3",
-    "assert": "^1.4.1",
-    "babel-eslint": "^6.1.2",
+    "@briancavalier/assert": "^3.4.0",
+    "@most/core": "^0.13.0",
+    "@most/types": "^0.11.1",
     "buba": "^3.0.0",
     "cpy-cli": "^1.0.1",
-    "eslint": "^3.3.1",
-    "eslint-config-standard": "^5.3.5",
-    "eslint-plugin-promise": "^2.0.1",
-    "eslint-plugin-standard": "^2.0.0",
     "flow-bin": "^0.54.0",
     "greenkeeper-postpublish": "^1.0.1",
     "istanbul": "^1.1.0-alpha.1",
-    "jsinspect": "^0.8.0",
     "mocha": "^3.0.2",
-    "most": "^1.0.1",
     "rollup": "0.40.0",
     "rollup-plugin-buble": "0.15.0",
+    "snazzy": "^7.0.0",
+    "standard": "^10.0.3",
     "uglify-js": "^2.7.1"
   },
-  "peerDependencies": {
-    "most": "^1.0.1"
+  "standard": {
+    "ignore": [
+      "test/flow"
+    ]
+  },
+  "dependencies": {
+    "@most/scheduler": "^0.13.0"
```

Comment:
Sorry, I missed this before.  Since the TS/flow types are distributed in the package, and they directly reference the Stream type from `@most/types`, I think `@most/types` has to be a dep rather than a devDep.